Create Entities of the diagram

Ygoraphael opened the pull request.

Commit:
Create Entities of the diagram

dangoth commented on Portal_Perfomance_Employes/Models/Certification.cs:
```diff
@@ -0,0 +1,16 @@
+        public int SkillId { get; set; }
+        public Skill Skill { get; set; }
+        public string CertificationName { get; set; }
+        public ICollection<EmployeeCertification> EmployeeCertifications { get; set; } = new List<EmployeeCertification>();
```
I don't think we should have the collection here, it should be enough to have it on the Employee

dangoth commented on Portal_Perfomance_Employes/Models/EmployeeCertification.cs:
```diff
@@ -0,0 +1,21 @@
+        public int Id { get; set; }
+        [ForeignKey("CertificationId")]
+        public int CertificationId { get; set; }
+        public Certification Certification { get; set; }
```
I think we just need the CertificationId here, if we want the certification we can retrieve it by ID

dangoth commented on Portal_Perfomance_Employes/Models/EmployeeCertification.cs:
```diff
@@ -0,0 +1,21 @@
+        public Certification Certification { get; set; }
+        [ForeignKey("Id")]
+        public int EmployeeId { get; set; }
+        public Employee Employee { get; set; }
```
same for employee

dangoth commented on Portal_Perfomance_Employes/Models/EmployeeCertification.cs:
```diff
@@ -0,0 +1,21 @@
+        [DataType(DataType.Date)]
+        [DisplayFormat(DataFormatString = "{0:yyyy/MM/dd}", ApplyFormatInEditMode = true)]
+        public DateTime DateCertified { get; set; }
+        public string CertifcateUrl { get; set; }
```
Certificate

Commit:
Change the name of tables to plural
Removed old migrations because changing names was not being applied to database

Commit:
Fix name property and create migration to change it on db
